seccomp_gen/cmd/falco_syscalls_formatter/main.go

```diff
@@ -0,0 +1,69 @@
+package main
+
+import (
+	"bufio"
+	"encoding/json"
+	"fmt"
+	"os"
+	"os/signal"
+	"syscall"
+)
+
+/*
+ * This binary takes the falco syscalls stream as the input (from stdio) and formats them and writes them to a file
+ * Requirements: 
+ * 1. Falco `json_output` has to be enabled
+ * 2. Falco rule should have `syscall.type` in the output
+*/
+
+type falcoOutput struct {
```

Comment:
The indentation here is a bit different than the code up or below, maybe try to use formatting tools like `gofmt`